statsmodels/genmod/generalized_estimating_equations.py

```diff
@@ -1094,6 +1117,11 @@ def fit(self, maxiter=60, ctol=1e-6, start_params=None,
         results.ctol = ctol
         results.maxiter = maxiter
 
+        if hasattr(self, "offset"):
+            results.offset = self.offset
+        if hasattr(self, "exposure"):
+            results.exposure = self.exposure
```

Comment:
is this needed/useful? we have them available through `results.model.xxx` 
I don't think we do it in other models. ??